### PR TITLE
Updated combine-metaphlan-humann tool

### PR DESCRIPTION
**Pull request by tuncK**
Old tool version is only compatible with humann2 and metaphlan < v3. The order and content of the output files from the two program versions differ. It should hopefully be able to process both types now.

### CODE REVIEW

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <description>to relate genus/species abundances and gene families/pathways abundances</description>
+    <requirements>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+        <exit_code range=":-1" />
+    </stdio>
+    <version_command></version_command>
+    <command><![CDATA[
+        python $__tool_directory__/combine_metaphlan_humann.py
```
Comment (bgruening) [inline tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:11]: Quotes

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            <filter>type=="gene_families"</filter>
+        </data>
+        <data name="pathway_output_file" format="tabular"
+            label="${tool.name} on ${on_string}: Pathway abundances related to genus/species abundances" >
+            <filter>type=="pathways"</filter>
+        </data>
+    </outputs>
+    <tests>
+       	<!-- Using the output of old (i.e. v2) Humann and metaphlan -->
+    	<test>
```
Comment (bgruening) [inline tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:42]: Expect num output

**Review by bgruening — COMMENTED** [review]
@bebatut can you have a look at this please.

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml (hunk 2 above)
Comment (tuncK) [reply tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:42]: Done

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<tool id="combine_metaphlan_humann" name="Combine MetaPhlAn and HUMAnN outputs" version="0.3.0" profile="22.01">
+    <description>to relate genus/species abundances and gene families/pathways abundances</description>
+    <requirements>
+    </requirements>
+    <stdio>
```
Comment (bgruening) [inline tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:5]: stdio can be removed in this form and replaced with "detect_error" on the command tag

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml [hunk 4]
```diff
@@ -0,0 +1,112 @@
+<tool id="combine_metaphlan_humann" name="Combine MetaPhlAn and HUMAnN outputs" version="0.3.0" profile="22.01">
+    <description>to relate genus/species abundances and gene families/pathways abundances</description>
+    <requirements>
```
Comment (bgruening) [inline tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:3]: no requirements? not even python?

Comment (tuncK) [reply tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:3]: It requires python to run, but the prev. version was OK with the python of the environment. Should I still add?

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml (hunk 3 above)
Comment (tuncK) [reply tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:5]: Done

File: tools/combine_metaphlan_humann/combine_metaphlan_humann.xml (hunk 4 above)
Comment (tuncK) [reply tools/combine_metaphlan_humann/combine_metaphlan_humann.xml:3]: Anyways, done.

**Review by bgruening — APPROVED** [review]
Ok, for me.

**Comment by bgruening** [comment]
@bebatut any comment from your side?

**Review by bebatut — COMMENTED** [review]
Just one suggestion. Otherwise looks good. Thanks @tuncK

**Comment by tuncK** [comment]
Thanks for your suggestion, @bebatut. Done.

**Pull request merged**